website/docs/Installation/linux-install.md

```diff
@@ -1,64 +1,131 @@
 ---
 sidebar_position: 5
+title: Linux
+description: You can install Podman Desktop on Linux from Flathub, a Flatpak bundle, or a ZIP archive.
+tags: [podman-desktop, installing, linux, flathub, flatpak]
+keywords: [podman desktop, podman, containers, installing, installation, linux, flathub, flatpak]
 ---
 
-# Linux
+# Installing Podman Desktop on Linux
 
-Check out the [Downloads](/downloads/linux) section of this website to download either the [.flatpak](https://flatpak.org/setup/Fedora) file or the zip file depending on your preference.
+You can install Podman Desktop from:
 
-To use Podman Desktop, the latest version of Podman is required. Podman is a rootless, daemonless container engine. Read more about it [here](https://podman.io/whatis.html).
+* [Flathub](#flathub)
+* [A Flatpak bundle](#flatpak-bundle)
+* [An archive](#archive)
 
-## Installing Podman Desktop
+## Installing Podman Desktop from Flathub {#flathub}
 
-### 1. Using [FlatHub](https://flathub.org/apps/details/io.podman_desktop.PodmanDesktop)
+Consider installing the Podman Desktop from Flathub to have:
 
-With FlatHub, you can install Podman Desktop with just a single command.
+* One command installation
+* Package updates
 
-> Follow the [setup guide](https://flatpak.org/setup/) before installing Podman Desktop
+#### Prerequisites
 
-```sh
-flatpak install --user flathub io.podman_desktop.PodmanDesktop
-```
-After installation, to open Podman Desktop, use the following command:
+* [Flatpak](https://flatpak.org/setup/)
+* [Podman](https://podman.io/whatis.html) stable version
 
-```sh
-flatpak run io.podman_desktop.PodmanDesktop
-```
+#### Procedure
 
-### 2. Using [Flatpak](https://flatpak.org/setup/) application from [Downloads](/downloads/linux)
+1. Assert the Flathub repository is enabled, and add it if required:
 
-:::infoPrerequisite 
-Verify Flatpak is set up in your distribution. Click [here](https://flatpak.org/setup/) to learn more about setting up Flatpak.
-:::
+    ```shell
+    $ flatpak remote-add --if-not-exists --user flathub https://flathub.org/repo/flathub.flatpakrepo
+    ```
 
-Open the Terminal and go to Downloads directory. 
+2. Install Podman Desktop from Flathub:
 
-```sh
-cd Downloads
-```
+    ```shell
+    $ flatpak install --user flathub io.podman_desktop.PodmanDesktop
+    ```
 
-> The Flatpak identifier for the application is `io.podman_desktop.PodmanDesktop`. This is required to install & start the application using Flatpak.
+#### Verification
 
-Run the following command to install the Flatpak application,
+* Open Podman Desktop from a terminal:
 
-```sh
-flatpak install io.podman_desktop.PodmanDesktop
-```
+  ```shell
```

Comment:
we should use `console` actually instead of shell?

Reply:
Learning ... https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js

Reply:
OK we need to extend : 
https://docusaurus.io/docs/markdown-features/code-blocks
https://prismjs.com/#supported-languages

Shell session - shell-session, sh-session, shellsession

Reply:
should be fine as today I just learned shell is an alias to console for markdown